modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -63,43 +63,45 @@ WARNING: You cannot insert documents into a SASL bucket if you have a read-only
 
 === RBAC Privileges
 
-User executing the INSERT statement must have the _Query Insert_ privilege on the target keyspace.
+To execute the INSERT statement, you must have the _Query Insert_ privilege on the target keyspace.
 
 If the statement has any SELECT or RETURNING data-read clauses, then the _Query Select_ privilege is also required on the keyspaces referred in the respective clauses.
 For more details about roles and privileges, see {authorization-overview}[Authorization].
 
+WARNING: Please note that the examples below will alter the data in your sample buckets.
+
 For example,
 
-* To execute the following statement, user must have the _Query Insert_ privilege on `pass:c[`travel-sample`]`.
+* To execute the following statement, you must have the _Query Insert_ privilege on `pass:c[`travel-sample`.inventory.hotel]`.
 +
 [source,n1ql]
 ----
-INSERT INTO `travel-sample` (KEY, VALUE)
-VALUES ("key1", { "type" : "hotel", "name" : "new hotel" })
+INSERT INTO `travel-sample`.inventory.hotel (KEY, VALUE)
+VALUES ("key1", { "type" : "hotel2", "name" : "new hotel scopes" });
 ----
 
-* To execute the following statement, user must have the _Query Insert_ privilege on `pass:c[`travel-sample`]` and the _Query Select_ privilege on `pass:c[`travel-sample`]`.
+* To execute the following statement, you must have the _Query Insert_ and _Query Select_ privileges on `pass:c[`travel-sample`.inventory.hotel]`.
 +
 [source,n1ql]
 ----
-INSERT INTO `travel-sample` (KEY, VALUE)
-VALUES ("key1", { "type" : "hotel", "name" : "new hotel" }) RETURNING *
+INSERT INTO `travel-sample`.inventory.hotel (KEY, VALUE)
+VALUES ("key1", { "type" : "hotel", "name" : "new hotel" }) RETURNING *;
 ----
 
-* To execute the following statement, user must have the _Query Insert_ privilege on `pass:c[`travel-sample`]` and _Query Select_ privilege on `pass:c[`beer-sample`]`.
+* To execute the following statement, you must have the _Query Insert_ privilege on `pass:c[`travel-sample`.inventory.hotel]` and _Query Select_ privilege on `pass:c[`beer-sample`]`.
 +
 [source,n1ql]
 ----
-INSERT INTO `travel-sample` (KEY foo, VALUE bar)
-SELECT foo, bar FROM `beer-sample`
```

Comment:
Not completely sure what we were trying to showcase here previously, this query doesn't seem to work. Any ideas?

The error is:
```
 "Cannot INSERT non-string key Missing field or index foo. of type value.missingValue."
 ```